Lib/fontTools/varLib/models.py

```diff
@@ -530,6 +531,22 @@ def interpolateFromMastersAndScalars(self, masterValues, scalars, *, round=noRou
         deltas = self.getDeltas(masterValues, round=round)
         return self.interpolateFromDeltasAndScalars(deltas, scalars)
 
+    @staticmethod
+    def interpolateFromMastersAndMasterScalars(masterValues, masterScalars):
```

Comment:
This function is literally the same as `interpolateFromDeltasAndScalars()`, except for the variable names. Do we really need both?